.gitignore

```diff
@@ -25,3 +25,7 @@ data/
 pkg/web/static/tailwind.css
 tmp/
 mentions.json
+node_modules/
+package.json
+package-lock.json
+.air.toml.local
```

Comment:
What's the purpose of `.air.toml.local`?

Reply:
Put context in the commit: 

```
- ignore .air.toml.local, on windows the built binary _has_ to be
  `.exe`, so i've created a different file on my machine that has
  `.exe` in it
```